### DISCUSSION TITLE
how to make a change when dark mode is activate or  light mode

### DISCUSSION THREAD

**Discussion by nuexq** [shadcn-ui/ui#2899] [General]
I want to change an icon when light mode is set and change it to other icon when dark mode is set 
how to do that

**Comment by pablotanner** [comment]
In tailwind.config.js you need to add the following:

```javascript
module.exports = {
  content: [
       .....
  ],
  darkMode: "class",
```

Then you can use the "dark:" selector of tailwind to do the following:

```javascript
<DarkIcon className="hidden dark:flex" />
<LightIcon className="dark:hidden" />
```

**Reply by nuexq** [reply]
thanks man i fix the problem with another method but this method is look more good
